py3status/modules/xrandr.py

```diff
@@ -399,11 +402,15 @@ def xrandr(self):
         self._set_available_combinations()
         self._choose_what_to_display()
 
-        if self.fixed_width is True:
-            output = self._center(self.displayed)
+        if (len(self.available_combinations) < 2 and
+                self.hide_if_single_combination):
+            full_text = ''
```

Comment:
Words hurts me! :sob:

From what I gather, @tobes want to epxnad more on the formatter (eventually) and we probably need more formatter options to make it to work with all modules so the users can tweak things using the formatter instead of adding more "if-else" options so the code can stay simple and raw as possible.

My first PR here was adding simple `hide_if_zero` option in `imap` which could be done with the formatter instead. I learn about it later. We really need some formatter documentation asap.

Because formatter just works with all modules, I think xrandr may need to pass along at least a "number of monitors" placeholder here to make formatter work with count or length, but it's more stuffs we need to figure out on first.


Instead of passing back `''`, we pass back nothing to `{output}` so we may get `{output}` in our status.

If we got nothing for `{output}`, we print `{output}`.
If we got nothing for `[{output}]`, we print  `''`.

I think we have to use `format = '[{output}]'` option here too. Thoughts? @tobes